src/templates/app/cashier/payment_methods.jsx

```diff
@@ -91,9 +91,9 @@ const PaymentMethods = () => {
 
     const deposit                  = 'Deposit: ';
     const withdrawal               = 'Withdrawal: ';
-    const working_day              = '[_1] working day';
+    const period                = '[_1] day';
```

Comment:
Could you keep the equal sign alignment? 😆 It is spaced out on purpose, so that all equal signs are on the same column.

Reply:
I thought there is smth to do it on precommit 😄